exercises/practice/lens-person/lens-person.spec.js

```diff
@@ -32,9 +32,7 @@ describe('nameLens', () => {
   });
 
   xtest('should ensure immutability by checking the original person object', () => {
-    expect(person).not.toStrictEqual(
-      new Person(new Name('Sara', 'Lakshamanan'), person.born, person.address),
-    );
+    expect(person.name).toStrictEqual(new Name('Saravanan', 'Lakshmanan'));
```

Comment:
This doesn't ensure immutability.
The original test was also not ensuring immutability.

Instead the test should check that `person.name` should not be changed when calling `nameLens.set`